src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+
+.card {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  font-style: normal;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  color: #060b35;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
check correct properties order
https://9elements.com/css-rule-order/

Reply:
still not fixed